### ISSUE TITLE
Navigator.nvim is broken in 1.2.0 release

### ISSUE THREAD

**Issue by nwaywood** [LunarVim/LunarVim#3973]
### Problem description

The navigator keybindings stopped working since I updated to the `release-1.2/neovim-0.8-8f4a7bde`

They still work in my [nvim config](https://github.com/nwaywood/dotfiles/blob/master/config/nvim/lua/user/plugins/navigator.lua) which has the exact same navigator config as my [lvim config](https://github.com/nwaywood/dotfiles/blob/master/config/lvim/lua/user/plugins/navigator.lua). And they are obviously both using the same [tmux config](https://github.com/nwaywood/dotfiles/blob/master/tmux/tmux.conf.symlink#L83-L88)

### LunarVim version

release-1.2/neovim-0.8-8f4a7bde

### Neovim version (>= 0.8.0)

NVIM v0.8.3 

### Operating system/version

macOS 13.2.1

### Steps to reproduce

1. Install and setup https://github.com/numToStr/Navigator.nvim
2. Open `lvim` inside tmux (I'm on the latest version tmux 3.3a)
3. Navigator keybindings don't work

**Comment by kobzar** [comment]
Chanhe row in tmux config
set -g default-terminal xterm-256color

**Comment by nwaywood** [comment]
@kobzar thanks for the reply!

I just tried that, unfortunately it doesn't fix the issue :(

**Comment by manofthelionarmy** [comment]
Hey, I just ran into this issue too. I think I fixed it. In your tmux conf, go down to the line of this:
```
not_tmux="ps -o state= -o comm= -t '#{pane_tty}' | grep -iqE '^[^TXZ ]+ +(\\S+\\/)?(g?(view|n?vim?x?)(diff)?|fzf)$'"
```

Far over to the right of that, there is:
```
...(g?(view|n?vim?x?)(diff)?|fzf)$'" 
```
I fixed it by changing it to this:
```
...(g?(view|(n|l)?vim?x?)(diff)?|fzf)$'" 
```
The way I understand it is that in the tmux conf, tmux was originally checking if the current process in the pane is either nvim or vim. Lunarvim kicks of the neovim process as `lvim`, Hence why we check if the prefix of `vim` is either `l` for `lvim` or `n` for `nvim` in the fix.
